app/controllers/inbox_controller.rb

```diff
@@ -0,0 +1,17 @@
+class InboxController < ApplicationController
+  include Mandrill::Rails::WebHookProcessor
+
+  def handle_inbound(event_payload)
+    handler = IncomingMail::Handler.new
+    resp = handler.handle(event_payload)
+    Rails.logger.warn(resp.inspect)
+    if resp.action == IncomingMail::Response::ERROR
+      # TODO do we want mandrill to keep trying on our error?
+      head 500
```

Comment:
Yep, we use New Relic to (automatically) track exceptions. I'd allow it to bubble up, and use [`rescue_from`](http://guides.rubyonrails.org/action_controller_overview.html#rescue-from) if you need some special handling of it.